### PR TITLE
feat:support quick insert command by contextmenu

### PR DESCRIPTION
**Pull request by smildlzj**
**Thanks for contributing to the Selenium IDE!**
**A PR well described will help maintainers to quickly review and merge it**

### Description

**support quick insert command by contextmenu.**

Video_2023-05-08_160855

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium-ide/blob/master/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by toddtarsi** [comment]
This already exists in v4

**Comment by toddtarsi** [comment]
Oh oops! I didn't realize this was a PR re-opening my mistake @smildlzj

**Comment by toddtarsi** [comment]
@smildlzj - I want to check beforehand, how far off is this context menu from what you're looking for?

Screenshot 2023-05-08 at 7 10 24 AM

This menu is defined here:

https://github.com/SeleniumHQ/selenium-ide/blob/f0bf2f258af9e4c934f32e36c2e68e599ebf296d/packages/selenium-ide/src/main/session/controllers/Menu/menus/testEditor.ts

And its consumed here:

https://github.com/SeleniumHQ/selenium-ide/blob/df42baa760a4947ed9ad2b5ce707c3fd8e1612c0/packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Suites/Editor/CurrentSuiteTestRow.tsx#L48

I want to point you towards this because its kinda extensible and it also helps support hot key bindings and that stuff.

**Comment by smildlzj** [comment]
> @smildlzj - I want to check beforehand, how far off is this context menu from what you're looking for?
> 
> 
> This menu is defined here:
> 
> https://github.com/SeleniumHQ/selenium-ide/blob/f0bf2f258af9e4c934f32e36c2e68e599ebf296d/packages/selenium-ide/src/main/session/controllers/Menu/menus/testEditor.ts
> 
> And its consumed here:
> 
> https://github.com/SeleniumHQ/selenium-ide/blob/df42baa760a4947ed9ad2b5ce707c3fd8e1612c0/packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Suites/Editor/CurrentSuiteTestRow.tsx#L48
> 
> I want to point you towards this because its kinda extensible and it also helps support hot key bindings and that stuff.

@toddtarsi   i think isnt same menu.  this pr of  menu will show on playback window  

image

**Comment by smildlzj** [comment]
@toddtarsi pls review

File: packages/selenium-ide/src/browser/windows/PlaybackWindow/preload/record-shortcuts.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { ipcRenderer } from 'electron'
+import Recorder from './recorder'
+import LocatorBuilders from './locator-builders'
+
+const locatorBuilders = new LocatorBuilders(window)
+
+let recorder: Recorder
+let eleTarget: HTMLElement | null
+
+const shortcutHandler = new Map
```
Comment (toddtarsi) [inline packages/selenium-ide/src/browser/windows/PlaybackWindow/preload/record-shortcuts.ts:10]: i have never seen this syntax before, wild

**Review by toddtarsi — APPROVED** [review]
I love it, LGTM 👍

File: packages/selenium-ide/src/browser/windows/PlaybackWindow/preload/record-shortcuts.ts (hunk 1 above)
Comment (smildlzj) [reply packages/selenium-ide/src/browser/windows/PlaybackWindow/preload/record-shortcuts.ts:10]: packages/selenium-ide/src/browser/windows/PlaybackWindow/preload/record-shortcuts.ts 
`record-shortcuts.ts ` is new file,  just handle click event from  main menu. and then send to `record` api

Comment (toddtarsi) [reply packages/selenium-ide/src/browser/windows/PlaybackWindow/preload/record-shortcuts.ts:10]: Yeah no, the constructor for map invoked without parens. Mind blown haha!

**Comment by toddtarsi** [comment]
It's approved. I did not realize you were unable to merge after your PR is approved. My mistake on workflow. I'm clicking merge now :)

**Pull request merged**